data-prepper-api/src/main/java/org/opensearch/dataprepper/metrics/PluginMetrics.java

```diff
@@ -78,7 +78,7 @@ public <T> T gauge(String name, T obj, ToDoubleFunction<T> valueFunction) {
         return Metrics.gauge(getMeterName(name), obj, valueFunction);
     }
 
-    private String getMeterName(final String name) {
+    public String getMeterName(final String name) {
```

Comment:
I don't see this being used here. Do we still need to expose this method?

Reply:
It is used in the tests. Some of the other tests were using local string concatenation to get this value. I figured there wasn't any harm in exposing this method since it's read-only and allows for a single source of truth in the formatting.

Let me know if you are strongly against exposing this method and I'll swap to local string concatenation.

Reply:
I'm not strongly opposed. I didn't see it used at first. But, it is because some diffs were hidden by GitHub. If you need it, I'm fine with adding it.

Reply:
I was to able to get rid of this change by moving the existing counter